test/evergreen.yml

```diff
@@ -4213,6 +4224,7 @@ buildvariants:
   - ubuntu2004-test
   tasks:
     - name: compatibility-test-for-newer-releases
+    - name: compatibility-test-upgrade-to-latest
```

Comment:
Putting the new task here will get it run for each commit build. Should we consider running it less frequently (by putting it into the `compatibility-tests-less-frequent` builder)?